api/v3/Dedupe.php

```diff
@@ -107,7 +107,8 @@ function civicrm_api3_dedupe_getstatistics($params) {
     $params['rule_group_id'],
     CRM_Utils_Array::value('group_id', $params),
     CRM_Utils_Array::value('criteria', $params, []),
-    CRM_Utils_Array::value('check_permissions', $params, [])
+    CRM_Utils_Array::value('check_permissions', $params, []),
+    CRM_Utils_Array::value('search_limit', $params, [])
```

Comment:
Do these fallbacks make sense? Feels like it should be `TRUE` or `FALSE` for `check_permissions` (not sure which, `[]` seems to have worked so far but that looks weird) and `0` for `search_limit`.